feat(curriculum): add cipher project to Python curriculum (#52373)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/20-upcoming-python/learn-string-manipulation-by-building-a-cipher/6555e1bfcd374e18c6be8e58.md

---
id: 6555e1bfcd374e18c6be8e58
title: Step 78
challengeType: 20
dashedName: step-78
---

# --description--

> In Python there's a way to easily format strings. *f-strings* enables you to interpolate values in your strings. Interpolation means to put placeholders to replace with specified values. For example, you can achieve the same result of `'Encrypted text: ' + text` with `f'Encrypted text: {text}'`. You need to put an "f" before the quotes to create the f-string and put the variables or expression you want to interpolate between curly braces.

Reviewer: ```suggestion
In Python, there's a way to easily format strings. *f-strings* enables you to interpolate values in your strings. Interpolation means to put placeholders to replace with specified values. For example, you can achieve the same result of `'Encrypted text: ' + text` with `f'Encrypted text: {text}'`. You need to put an "f" before the quotes to create the f-string and put the variables or expressions you want to interpolate between curly braces.

```

---
id: 6555e1bfcd374e18c6be8e58
title: Step 87
challengeType: 20
dashedName: step-87
---

# --description--

> In Python, there's a way to easily format strings. *f-strings* enables you to interpolate values in your strings. 

Reviewer 2: ```suggestion
In Python, there's a way to easily format strings. *f-strings* enable you to interpolate values in your strings. 
```

> Interpolation means to put placeholders to replace with specified values. For example, you can achieve the same result of `'Encrypted text: ' + text` with `f'Encrypted text: {text}'`. You need to put an `f` before the quotes to create the f-string and put the variables or expressions you want to interpolate between curly braces.

Reviewer 2: ```suggestion
Interpolation means writing placeholders that will be replaced by the specified values when the program runs. For example, you can get the same result of `'Encrypted text: ' + text` with `f'Encrypted text: {text}'`. You need to put an `f` before the quotes to create the f-string and write the variables or expressions you want to interpolate between curly braces.
```

> Modify the first `print` call to use an f-string.

Reviewer 2: ```suggestion
Modify the first `print()` call to use an f-string.
```

---
id: 6555e1bfcd374e18c6be8e58
title: Step 87
challengeType: 20
dashedName: step-87
---

# --description--

> In Python, there's a way to easily format strings. *f-strings* enabls you to interpolate values in your strings. 

Reviewer 3: ```suggestion
In Python, there's a way to easily format strings. *f-strings* enables you to interpolate values in your strings. 
```

> In Python, there's a way to easily format strings. *f-strings* enables you to interpolate values in your strings. 

Reviewer 4: ```suggestion
In Python, there's a way to easily format strings. *f-strings* enabls you to interpolate values in your strings. 
```

> Interpolation means to put placeholders to replace with specified values. For example, you can achieve the same result of `'Encrypted text: ' + text` with `f'Encrypted text: {text}'`. You need to put an "f" before the quotes to create the f-string and put the variables or expressions you want to interpolate between curly braces.

Reviewer 4: ```suggestion
Interpolation means to put placeholders to replace with specified values. For example, you can achieve the same result of `'Encrypted text: ' + text` with `f'Encrypted text: {text}'`. You need to put an `f` before the quotes to create the f-string and put the variables or expressions you want to interpolate between curly braces.
```
